pkg/roachprod/install/cockroach.go

```diff
@@ -520,8 +520,9 @@ func (c *SyncedCluster) LogDir(node Node, virtualClusterName string, instance in
 	return dirName
 }
 
-// InstanceStoreDir returns the data directory for the given instance of
-// the given virtual cluster.
+// InstanceStoreDir returns the data directory for the given instance of the
+// given virtual cluster. This should only be used for external processes. For
+// shared processes to get the data directory, use NodeDir.
 func (c *SyncedCluster) InstanceStoreDir(
```

Comment:
Maybe rename to `TenantInstanceStoreDir`? I still find it confusing that in the default case it's `/mnt/data%d/cockroach`, and in this case it's `data%s`.

Reply:
Yes, `TenantInstanceStoreDir` would be more descriptive. The default case (NodeDir) is for storage of the system nodes (or storage nodes). This store dir for an external process is light afaik, stores no actual db data. The reason it's a string is to include the tenant name, and instance # (multiple instances can be spawned for the same tenant).